test/github-trigger

```diff
@@ -16,6 +16,10 @@ def main():
 
     github = testinfra.GitHub()
 
+    if opts.context not in testinfra.DEFAULT_VERIFY:
```

Comment:
This sounds like a good place to check for `--force`. With it, we should emit a warning but still go ahead.